docs: make structural directives guide generic (#44895)
angular/angular: aio/content/special-elements/core/ng-template.md

With `<ng-template>`, you can define template content that is only being rendered by Angular when you, whether directly or indirectly, specifically instruct it to do so, allowing you to have full control over how and when the content is displayed.

<div class="alter is-helpful">

>   Note that if you simply wrap content inside an `<ng-template>` without instructing Angular to render it, such content will simply not appear in your application. As you can see in the following example, Angular does not render the middle "Hip!" in the phrase "Hip! Hip! Hooray!" because of the surrounding `<ng-template>`.

Reviewer: ```suggestion
  Note that if you wrap content inside an `<ng-template>` without instructing Angular to render it, such content will not appear on a page. As you can see in the following example, Angular does not render the middle "Hip!" in the phrase "Hip! Hip! Hooray!" because of the surrounding `<ng-template>`.
```

Author: I've updated the text, I've also noticed that "As you can see" part which actually made sense before when there was an image but doesn't anymore, so I restructured that sentence a bit :slightly_smiling_face:
